.github/workflows/run_tests.yml

```diff
@@ -59,49 +63,36 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-20.04] # cannot test on macOS as docker isn't supported on Mac
-        compiler: [intel, 8, 9, 10, 11] # intel compiler, and versions of GNU compiler
-        rai_v: [1.2.4, 1.2.5] # versions of RedisAI
+        rai_v: [1.2.5] # versions of RedisAI
```

Comment:
Was it intentional to drop support for RedisAI 1.2.4 as part of this PR? I agree it needs to be done, but that seems like a separate issue than addressing code coverage

Reply:
Thanks for the catch, this was an inadvertent change to limit testing while developing